deployments/examples/spark30/cmd/run.sh

```diff
@@ -16,12 +16,40 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# replace these env var values accordingly
-SPARK_HOME=/Users/example/repository/spark
+SCRIPT_PATH=$(cd `dirname $0`; pwd)
+# set up root directory
+WORK_SPACE_ROOT=`dirname $SCRIPT_PATH`
+
+SPARK_VERSION=3.0.0
+SPARK_HADOOP_VERSION=2.7
```

Comment:
See below, since you do not download the checksum file this is really still fixed to one specific version.
We should also be able to pick up these values from outside the script, for both spark and hadoop. Something like this:
```
if [ "$SPARK_VERSION" = "" ]; then
    echo using default Spark version;
    SPARK_VERSION=3.0.0
fi
```

Reply:
But we have `spark24` demo dir. Maybe we can merge these two isolated demos through variables?

Reply:
Sure that would be nice, just one version to maintain.